Update BUILD.md (#621)
cemu-project/Cemu: BUILD.md

To compile Cemu, a recent enough compiler and STL with C++20 support is required! LLVM 13 and 
below, built in LLVM, and Xcode LLVM don't support the C++20 feature set required. The OpenGL graphics

> API isn't supported on macOS, so Vulkan must be used. Additionally, Vulkan must be used through the 

Reviewer: ```suggestion
API is not new enough on macOS, so Vulkan must be used. Additionally, Vulkan must be used through the 
```
